Readme.md

```diff
@@ -1,9 +1,12 @@
 # CRAPLang - CSS Really is a Programming Language
+![Developed by Badge](https://img.shields.io/badge/Dveloped_by-Shreyas-blue?logo=windowsterminal)
+![Powerd by Badge](https://img.shields.io/badge/Powerd_by-golang-orange?logo=go)
+
 > not to be confused with crablang, which is a completely different thing.
 
-CRAP is a general purpose programming language inspired by CSS.
+CRAP is a general purpose programming language inspired by CSS🔥.
 
-## Getting Started
+## Getting Started💫
 - No Comments, who needs them anyway. just write self-documenting code.
 - Selectors are used to define functions.
 - Attributes are used to define parameters.
```

Comment:
This is incorrect. This is supposed to be attreibutes, please, kind sir. Fix please. Thank you.